CRC.adoc

```diff
@@ -1,80 +1,278 @@
-+
-`export QUAY_NAMESPACE=<quay-username>`
+Extract the downloaded file into a directory of your choice:
 
-6. Ensure that the following Quay repositories are set to public:
-+
-* host-operator
-* member-operator
-* member-operator-webhook
-* registration-service
+[source,bash]
+----
+tar -xvf crc-linux-amd64.tar.xz
+----
 
-You can find these on the https://quay.io/repository/[Repository] page at quay.io:
+Give execution permissions to the binary and move it to a directory on your path or `/usr/local/bin`:
 
-image::quay_repo.png[align=center]
+[source,bash]
+----
+chmod u+x crc
+sudo mv crc /usr/local/bin
+----
 
-Click on the repository name, then click on the settings icon:
+=== Set up the cluster and enable or tweak the cluster's settings
 
-image::quay_repo_detail.png[align=center]
+You need to set up the OpenShift local cluster — the daemons, configurations and basic settings for it to be able to
+run — by running the command below. You only need to do it the first time you're setting up the cluster or after
+running `crc cleanup`:
 
-Scroll down to the *Repository Visibility* section, and ensure the repository is set to public:
+[source,bash]
+----
+crc setup
+----
 
-image::quay_repo_visibility.png[align=center]
+Also, in order to run the tests seamlessly and without any problems, there is a set of settings you are advised to
+change:
 
-7. Start the CRC container by executing `crc start`. This will take a few minutes to complete, and the output should look something like this:
+[source,bash]
+----
+# Cluster monitoring is required for the tests to pass
+crc config set enable-cluster-monitoring true
 
-image::crc_start_output.png[align=center]
+# A minimum of 14GB of virtual machine memory is required to run by default, so it is recommended to change that
+# setting. Tweaking the CPUs and disk size is optional, but also recommended to avoid having issues down the line.
+crc config set cpus 8
+crc config set disk-size 50
+crc config set memory 14500 # or 20000 for 20GB
+----
 
-Take note of the `oc login...` command shown in the output.
+Now you can go ahead and start the cluster. The first time you will need to provide the pull secret you were presented
+with in the OpenShift local download page — if you didn't grab it you can come back and copy it. Run the following
+command to start the cluster:
 
-8. Execute the following command to set up the necessary environment variables:
+[source,bash]
+----
+crc start
+----
 
-`eval $(crc oc-env)`
+While your local OpenShift cluster boots, you can go ahead and prepare the Quay repositories and the local ones to be
+able to run the tests.
 
-9. Log into the cluster by executing the `oc login` command as shown in the resulting output from running `crc start` in the previous step.  The output should look something like this:
-+
-image::crc_oc_login.png[align=center]
+=== Prepare the projects
+==== Creating the Quay repositories and making them public
```

Comment:
this doesn't seem to be CRC-specific steps. Also, we already have this doc where we list quay repos and what do do with them: https://github.com/codeready-toolchain/toolchain-e2e/blob/58301e81b3adfc88a5119f825b546abecf77e15e/quay.adoc